webapi/webapi-webspeech-w3c-tests/webspeech/SpeechGrammarList_addFromString.html

```diff
@@ -37,11 +37,11 @@
 
 test(function () {
     var grammarList, returnValue;
-    grammarList = new webspeechGrammarList();
+    grammarList = new webkitSpeechGrammarList();
```

Comment:
Hmm, prefer support both the standard API and API with webkit- prefix. See

https://github.com/w3c/web-platform-tests/blob/master/common/vendor-prefix.js
https://github.com/w3c/web-platform-tests/blob/master/vibration/cancel-when-hidden-manual.html